src/root_files/_common/404.html

```diff
@@ -99,9 +99,7 @@
     }
 
     var all_lang_regex = new RegExp('(' + all_languages.join('|') + ')', 'i');
-    var is_from_404 = location.search &&
```

Comment:
I removed `is_from_404` mechanism, since it wasn't necessary anymore. If this is crucial to the app, please let me know.